docs/reference/errors-and-warnings/NU1100.md

```diff
@@ -17,11 +17,16 @@ f1_keywords:
 <pre>Unable to resolve 'Dependency (>= 1.0.0)' for 'TargetFramework'</pre>
 
 ### Issue
-A dependency could not be resolved. This is a generic issue for types that are not packages or projects.
+A dependency could not be resolved. This is a generic issue for types that are not packages or projects. 
+
+This issue may also be caused by the lack of a NuGet configuration file.

```

Comment:
NuGet would normally be creating the user-wide configuration file if it doesn't already exist. 

Can you elaborate how you hit this?